runtime/interpreter/value.go

```diff
@@ -639,6 +648,74 @@ func (v BoolValue) Equal(_ *Interpreter, _ LocationRange, other Value) bool {
 	return bool(v) == bool(otherBool)
 }
 
+func (v BoolValue) Less(interpreter *Interpreter, other ComparableValue, locationRange LocationRange) BoolValue {
+	o, ok := other.(BoolValue)
+	if !ok {
+		panic(InvalidOperandsError{
+			Operation: ast.OperationLess,
+			LeftType:  v.StaticType(interpreter),
+			RightType: other.StaticType(interpreter),
+		})
+	}
+
+	if v {
```

Comment:
As you mentioned in the description/TODOs, the PR still needs interpreter test cases

Reply:
Added them in `TestInterpretComparison` in `runtime/tests/interpreter/interpreter_test.go`.